File: iosApp/iosAppTests/UIComponent/DebugMenu/FeatureFlagsDebug/FeatureFlagsDebugStateRobot.swift
#if DEBUG
import ComposeApp

final class FeatureFlagsDebugStateRobot: BaseStateRobot<FeatureFlagsDebugUiState, FeatureFlagsDebugEvent> {

    override func defaultState() -> FeatureFlagsDebugUiState {
        FeatureFlagsDebugUiState(rows: [], eventSink: createEventSink())
    }
}
#endif

File: features/debug-menu/impl/presentation/src/androidDeviceTest/kotlin/com/jkjamies/sampleplatter/features/debugmenu/presentation/FeatureFlagsDebugStateRobot.kt
package com.jkjamies.sampleplatter.features.debugmenu.presentation

import com.jkjamies.sampleplatter.core.test.StateRobot

class FeatureFlagsDebugStateRobot : StateRobot<FeatureFlagsDebugUiState, FeatureFlagsDebugEvent>() {

    override fun defaultState() = FeatureFlagsDebugUiState(
        rows = emptyList(),
        eventSink = createEventSink(),
    )
}
